frontend/src/features/admin-form/create/builder-and-design/BuilderAndDesignDrawer/FieldListDrawer/field-panels/BasicFieldPanel.tsx

```diff
@@ -61,5 +98,158 @@ export const BasicFieldPanel = ({ searchValue }: { searchValue: string }) => {
         </Box>
       )}
     </Droppable>
+  ) : (
+    <>
+      <Droppable isDropDisabled droppableId={CREATE_FIELD_FREE_TEXT_DROP_ID}>
+        {(provided) => (
+          <Box ref={provided.innerRef} {...provided.droppableProps}>
+            {filteredCreateBasicFreeTextFields.length > 0 && (
+              <FieldSection label="Free text">
+                {filteredCreateBasicFreeTextFields.map(
+                  ({ fieldType, originalIndex }) => (
+                    <DraggableBasicFieldListOption
+                      index={originalIndex}
+                      isDisabled={isLoading}
```

Comment:
This is some what inconsistent in the other field sections.

Other sections will maps it as `shouldDisableField`.

```ts
const shouldDisableField = isLoading
...
return (
  <DraggableBasicFieldListOption
    isDisabled={shouldDisableField}
    ...
  >
```